Core/src/org/sleuthkit/autopsy/report/modules/taggedhashes/SaveTaggedHashesToHashDbConfigPanel.java

```diff
@@ -79,13 +76,13 @@ public void mousePressed(MouseEvent evt) {
                 JList<?> list = (JList) evt.getSource();
                 int index = list.locationToIndex(evt.getPoint());
                 if (index > -1) {
-                    String value = tagsNamesListModel.getElementAt(index);
-                    tagNameSelections.put(value, !tagNameSelections.get(value));
+                    TagName tagName = tagsNamesListModel.getElementAt(index);
+                    tagNameSelections.put(tagName, !tagNameSelections.get(tagName));
```

Comment:
Could this be changed to `Objects.equal(tagNameSelections.get(tagName), true)` to avoid a possible NPE?

Reply:
This code is a little weird and needs more clean up than I did.  But its not possible to have an NPE, tagNameSelections and tagsNamesListModel are populated from the same list.  Eventually if someone has more time I think all the extra lists include TagNameSelections could be removed.